docs/topics/migrations.txt

```diff
@@ -276,6 +276,14 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
+
+.. note::
+
+    You may add an ``initial = True`` class property to the ``Migration`` class
+    to have Django apply the "initial migration" logic when the
```

Comment:
I think perhaps we may want a clearer description here than "apply the initial migration logic." Perhaps something like `... to have Django fake this migration if it finds its added tables or columns already present in the database when ...` ?